WalletWasabi.Fluent/ViewModels/Wallets/LoadingViewModel.cs

```diff
@@ -25,19 +26,35 @@ public LoadingViewModel(IWalletModel wallet)
 	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
 	{
 		_wallet.Loader.Progress
-					  .Do(p => UpdateStatus(p.PercentComplete, p.TimeRemaining))
+					  .Do(p => UpdateStatus(p.RemainingFiltersToDownload, p.CurrentHeight, p.ChainTip, p.Percent))
 					  .Subscribe()
 					  .DisposeWith(disposables);
 	}
 
-	private void UpdateStatus(double percent, TimeSpan remainingTimeSpan)
+	private void UpdateStatus(uint remainingFiltersToDownload, uint currentHeight, uint chainTip, double percentProgress)
 	{
-		Percent = percent;
-		var percentText = $"{Percent}% completed";
-
-		var userFriendlyTime = TextHelpers.TimeSpanToFriendlyString(remainingTimeSpan);
-		var remainingTimeText = string.IsNullOrEmpty(userFriendlyTime) ? "" : $"- {userFriendlyTime} remaining";
-
-		StatusText = $"{percentText} {remainingTimeText}";
+		if (remainingFiltersToDownload > 0)
+		{
+			StatusText = $"Downloading {remainingFiltersToDownload:N0} filters";
+			return;
+		}
+
+		Percent = percentProgress;
+
+		var remainingBlocks = chainTip - currentHeight;
+		var hoursRemaining = remainingBlocks / 6.0m;
+
+		// Convert hours to more readable format
+		var remainingTimeString = hoursRemaining switch
+		{
+			< 1 => "less than 1 hour",
+			< 24 => $"{Math.Ceiling(hoursRemaining)} hours",
+			< 720 => $"{Math.Ceiling(hoursRemaining / 24)} days",
+			< 8760 => $"{Math.Ceiling(hoursRemaining / 720)} months",
+			_ => $"{Math.Ceiling(hoursRemaining / 8760)} years"
+		};
+
+		StatusText = $"Synchronized: {currentHeight:N0} / {chainTip:N0}";
```

Comment:
Add `blocks` or `filters` to make this more clear.

```suggestion
		StatusText = $"Synchronized: {currentHeight:N0} / {chainTip:N0} blocks";
```